### PR TITLE
Add webhooks plugins

### PR DESCRIPTION
**Pull request by adamruzicka**
requires:
- [x] https://github.com/theforeman/puppet-foreman/pull/920
- [x] https://github.com/theforeman/puppet-foreman_proxy/pull/651

### CODE REVIEW

**Comment by adamruzicka** [comment]
[test foreman-installer]

**Review by ekohl — APPROVED** [review]

File: config/katello.migrations/20210304102446_add_proxy_shellhooks_plugin.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+answers['foreman_proxy::plugin::shellhooks'] ||= false
```
Comment (ehelms) [inline config/katello.migrations/20210304102446_add_proxy_shellhooks_plugin.rb:1]: I think you also want to add a migration for foreman-proxy-content.migrations and add it to the answers file for that scenario as well.

Comment (adamruzicka) [reply config/katello.migrations/20210304102446_add_proxy_shellhooks_plugin.rb:1]: Good point, added

**Comment by ehelms** [comment]
[test foreman-installer]

**Comment by ehelms** [comment]
Some of the failures seen here are due to some job updates:

 1) See https://github.com/theforeman/jenkins-jobs/pull/32

 2) There are some modules that are not Puppet 7 compatible and cause a failure (See below). We can drop Puppet 7 from the job for now or ignore it when analyzing PRs. 

```
Using /usr/local/rvm/gems/ruby-2.7.0 with gemset foreman-installer-pr-test-139-2-7-7-0

2021-03-25 19:43:29 [ERROR ] [root] Puppet 7.5.0 does not meet requirements for herculesteam-augeasproviders_core (>= 5.0.0 < 7.0.0)

2021-03-25 19:43:29 [ERROR ] [root] Incompatible version of Puppet used, cannot continue

Puppet 7.5.0 does not meet requirements for herculesteam-augeasproviders_core (>= 5.0.0 < 7.0.0)

Cannot continue due to incompatible version of Puppet. Use --skip-puppet-version-check to disable this check.

script returned exit code 30
```

**Comment by ehelms** [comment]
@ekohl could you weigh in here on which direction we should proceed?

**Comment by ekohl** [comment]
I'm leaning to ignoring for now. While it's not great, it may be a good motivation to support it.

**Comment by ehelms** [comment]
[test foreman-installer]

**Comment by ehelms** [comment]
Puppet 7 tests failing as expected

**Pull request merged**